deepmd/pt/model/model/make_model.py

```diff
@@ -469,6 +469,11 @@ def get_nnei(self) -> int:
             """Returns the total number of selected neighboring atoms in the cut-off radius."""
             return self.atomic_model.get_nnei()
 
+        @torch.jit.export
+        def get_ntypes(self) -> int:
```

Comment:
Not all models are made from the atomic model. This method should be added into the BaseModel.